ontology/cheminf.md

```diff
@@ -25,5 +25,18 @@ publications:
   title: 'The chemical information ontology: provenance and disambiguation for chemical data on the biological semantic web'
 repository: https://github.com/semanticchemistry/semanticchemistry
 tracker: https://github.com/semanticchemistry/semanticchemistry/issues
+usages:
+- description: ChEMBL uses CHEMINF in the RDF download
+  examples:
+  - description: Physicochemical properties are represented as classes that are typed with CHEMINF classes
+    url: http://rdf.ebi.ac.uk/resource/chembl/molecule/CHEMBL6939#mw_freebase
+  user: https://www.ebi.ac.uk/chembl/
+- description: ENM uses CHEMINF for chemical information terms
```

Comment:
Unfortunately, we have decided that usages must be proven by a link to a website that lists an individual code from the ontology (like your pubchem example). An ontology re-using another does not count as usage in the sense of this metadata entry.

Reply:
Yeah, saw it did not count; I did not read that as you cannot list it at all :)

Reply:
@matentzn, what about SPARQL endpoints? Do those count?

Reply:
Okay, let me know.